app/controllers/users_controller.rb

```diff
@@ -8,30 +8,27 @@ class UsersController < ApplicationController
   before_action :authenticate_user!, only: %i[onboarding_update onboarding_checkbox_update]
   rescue_from Errno::ENAMETOOLONG, with: :log_image_data_to_datadog
 
-  DEFAULT_FOLLOW_SUGGESTIONS = %w[ben jess peter maestromac andy liana].freeze
+  DEFAULT_FOLLOW_SUGGESTIONS = SiteConfig.suggested_users
+  INDEX_ATTRIBUTES_FOR_SERIALIZATION = %i[id name username summary profile_image].freeze
+  private_constant :INDEX_ATTRIBUTES_FOR_SERIALIZATION
 
   def index
     if !user_signed_in? || less_than_one_day_old?(current_user)
-      @users = User.where(username: DEFAULT_FOLLOW_SUGGESTIONS)
+      # Theoretically, no user should get to this point since they should
+      # already be signed in, but this is here as a safeguard in case they do.
```

Comment:
I truly couldn't find a scenario when a user could hit this endpoint since we should be checking that a user is signed in via the `ApplicationController`. I added this note for more context, because otherwise, this feels a bit confusing (at least, it was for me!) 😕

Reply:
Hmmmmmmmmm since we don't call `authenticate_user!` on this action someone could get here but I think this is unnecessary and the dirty remnants of some past behavior and a quick fix.

Reply:
So we could remove this IMO

Reply:
Cool, I'm definitely a fan of removing this and the test I added for it entirely 👍